core/src/main/java/com/radixdlt/messaging/core/MessageDispatcher.java

```diff
@@ -101,6 +100,11 @@ class MessageDispatcher {
   private final PeerManager peerManager;
   private final Addressing addressing;
 
+  private final RejectionCountingRateLimiter ttlExpiredLogRateLimiter =
+      new RejectionCountingRateLimiter(0.05);
+  private final LRUCache<NodeId, RejectionCountingRateLimiter> sendErrorLogRateLimitersByReceiver =
+      new LRUCache<>(100);
```

Comment:
Might be worth making it 150 so it's a little larger than the validator set (and can cover validators and seed nodes at Olympia launch)